code/src/index.css

```diff
@@ -1,13 +1,30 @@
 body {
+  height: auot;
```

Comment:
typo here. Change it to auto :)

Reply:
oh! thank you for seeing that jespa374! fixing it now.